Documentation/operations/upgrade.rst

```diff
@@ -351,6 +351,8 @@ Deprecated Options
   native-routing mode, set ``routing-mode=native`` (previously
   ``tunnel=disabled``). To configure the tunneling protocol, set
   ``tunnel-protocol=geneve`` (previously ``tunnel=geneve``).
+* The ``disable-cnp-status-updates``, ``cnp-node-status-gc-interval duration`` and ``enable-k8s-event-handover``
+  options are deprecated and will be removed in v1.15.
```

Comment:
What are they replaced with or what should users use instead? (You explain the motivation in the commit description, but I wouldn't expect users of this section to check it out.)